library/src/test/java/com/chuckerteam/chucker/api/ChuckerInterceptorTest.kt

```diff
@@ -496,5 +498,48 @@ internal class ChuckerInterceptorTest {
         assertThat(transaction.requestPayloadSize).isEqualTo(request.body!!.contentLength())
     }
 
+    @ParameterizedTest
+    @EnumSource(value = ClientFactory::class)
+    fun oneShotRequestBody_isNotAvailableToChucker(factory: ClientFactory) {
+        server.enqueue(MockResponse())
+        val client = factory.create(chuckerInterceptor)
+
+        val oneShotRequest = object : RequestBody() {
+            private val content = Buffer().writeUtf8("Hello, world!")
+            override fun isOneShot() = true
+            override fun contentType() = "text/plain".toMediaType()
+            override fun writeTo(sink: BufferedSink) {
+                content.readAll(sink)
+            }
+        }.toServerRequest()
+
+        client.newCall(oneShotRequest).execute().readByteStringBody()
+
+        val transaction = chuckerInterceptor.expectTransaction()
+        assertThat(transaction.isRequestBodyPlainText).isFalse()
```

Comment:
Do we need to do 2 assertions in this test? This assertion looks to me more like checking of request body type detection works fine.

Reply:
Yes, it shouldn't be here. It is a leftover from splitting #527. I'll remove it.